scio-repl/README.md

```diff
@@ -0,0 +1,167 @@
+Scio REPL
+=========
+
+# Getting started
+
+The Scio REPL is an extension of the Scala REPL, with added functionality that allows you to
+interactively experiment with Scio. Think of it as a playground to try out things.
+
+# Quick start
+
+The easiest way to start with Scio REPL is to assembly jar and run it:
+
+```bash
+$ sbt 'project scio-repl' assembly
+$ java -jar scio-repl/target/scala-2.11/scio-repl*-fat.jar
+Starting up ...
+Scio context is available at 'sc'
+Welcome to Scio REPL!
+scio>
+```
+Scio context is available as `sc`, this is the starting point, use `tab` completion, history and
+all the other goods of REPL to play around in local mode.
+
+# Tutorial
+
+Start simple pipeline from
+
+```
+scio> val noTwoS = sc.parallelize(List(1,2,3)).filter( _ != 2 ).map( "I like " + _ ).saveAsTextFile("/tmp/hate-2s")
+twoS: scala.concurrent.Future[com.spotify.scio.io.Tap[String]] = scala.concurrent.impl.Promise$DefaultPromise@270ab7bc
+scio> val result = sc.close
+[main] INFO com.google.cloud.dataflow.sdk.runners.DirectPipelineRunner - Executing pipeline using the DirectPipelineRunner.
+[main] INFO com.google.cloud.dataflow.sdk.runners.DirectPipelineRunner - Pipeline execution complete.
+result: com.spotify.scio.ScioResult = com.spotify.scio.ScioResult@220a5163
+scio> result.state
+res20: com.google.cloud.dataflow.sdk.PipelineResult.State = DONE
+scio> noTwoS.waitForResult().value.foreach(println)
+I like 3
+I like 1
+```
+
+To create Scio context connected to Google's Dataflow service pass Scio arguments on REPL startup -
```

Comment:
To create a ScioContext for Google Cloud Dataflow service pass Dataflow pipeline options ...?